src/components/InRequest/InRequestViewCompact.tsx

```diff
@@ -0,0 +1,126 @@
+
+  // Function used to display the Employee Type in a shortened format.
+  // If it is a Civilian add New/Existing after depending on the selection
+  const displayEmpType = (): string => {
+    let displayValue = "";
+    switch (formData.empType) {
+      case EMPTYPES.CIV:
+        displayValue =
+          "Civilian - " + (formData.isNewCiv === "yes" ? "New" : "Existing");
+        break;
+      case EMPTYPES.MIL:
+        displayValue = "Military";
+        break;
+      case EMPTYPES.CTR:
+        displayValue = "Contractor";
+        break;
+    }
+    return displayValue;
+  };
+
+  /* Callback function to be provided to the EditPanel component for action on Save/Cancel*/
+  const onEditSaveCancel = (formEdits: IInForm | undefined): void => {
+    // If the form was edited, then update the formData with this new data, otherwise no action needed
+    if (formEdits) setFormData(formEdits);
+  };
+
+  return (
+    <>
+      <div id="inForm" className={classes.compactContainer}>
+        <div>
+          <Label weight="semibold" htmlFor="empNameId">
+            Employee Name:
+          </Label>
+          <br />
+          <Text id="empNameId">{formData.empName}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="empTypeId">
+            Employee Type
+          </Label>
+          <br />
+          <Text id="empTypeId">{displayEmpType}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="gradeRankId">
+            Grade/Rank
+          </Label>
+          <br />
+          <Text id="gradeRankId">{formData.gradeRank}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="workLocationId">
+            Local or Remote?
+          </Label>
+          <br />
+          <Text id="workLocationId">{formData.workLocation}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="arrivalDateId">
+            Select estimated on-boarding date
+          </Label>
+          <br />
+          <Text id="arrivalDateId">{formData.eta?.toLocaleDateString()}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="officeId">
+            Office
+          </Label>
+          <br />
+          <Text id="officeId">{formData.office}</Text>
+        </div>
+        <div>
+          <Label weight="semibold" htmlFor="supGovLeadId">
+            Supervisor/Government Lead
+          </Label>
+          <br />
+          <Text id="supGovLeadId">{formData.supGovLead?.text}</Text>
+        </div>
+        {formData.empType === EMPTYPES.CIV && formData.isNewCiv === "no" && (
+          <div>
+            <Label weight="semibold" htmlFor="prevOrgId">
+              Previous Organization
+            </Label>
+            <br />
+            <Text>{formData.prevOrg}</Text>
+          </div>
+        )}
+      </div>
+      <InRequestEditPanel
+        formData={formData}
+        onEditSaveCancel={onEditSaveCancel}
+      />
```

Comment:
Going to read the article you sent about React Rendering and maybe it will help me understand how to best layer things.

Reply:
After reading the article you sent, it clicked. Props don't force a re-render, state changes do. If I move the component up a level it then becomes a child of the component I am updating the state of (instead of a sibling). So the changed state at the parent will then force the children to re-render too -- making the changed props be applied.